### PR TITLE
feat(kit): allow customising logger options

### PR DESCRIPTION
**Pull request by DaniAcu**
### 🔗 Linked issue

This issues reported in unjs/consola and nuxt-modules/tailwindcss. Show that developers are struggleling with the possibility to pass level configuration to their plugins. I check in consola repository but everything is okay from their side, but our kit is not exposing a clear way to customize the logger.

https://github.com/unjs/consola/issues/253
https://github.com/nuxt-modules/tailwindcss/pull/732

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

Currently this implementaion only allowed to pass options when you are using a tag.

```ts
useLogger("tag", { level: 0 });
```

But is not allowed to passing options configuration only.

```ts
useLogger({ level: 0 });
```

We can do it if you consider it as a better option.

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Atinux** [comment]
I like the idea 🚀

**Review by pi0 — APPROVED** [review]
Nice idea!

File: docs/3.api/5.kit/13.logging.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Returns a logger instance. It uses [consola](https://github.com/unjs/consola) un
 ### Type
 
 ```ts
-function useLogger (tag?: string): ConsolaInstance
+function useLogger (tag?: string, options?: Partial<ConsolaOptions>) : ConsolaInstance
```
Comment (danielroe) [inline docs/3.api/5.kit/13.logging.md:20]:
```suggestion
function useLogger (tag?: string, options?: Partial<ConsolaOptions>): ConsolaInstance
```

File: packages/kit/src/logger.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import { consola } from 'consola'
+import type { ConsolaOptions } from 'consola';
 
 export const logger = consola
 
-export function useLogger (tag?: string) {
-  return tag ? logger.withTag(tag) : logger
+export function useLogger (tag?: string, options: Partial<ConsolaOptions> = {}) {
+  return tag ? logger.create(options).withTag(tag) : logger;
```
Comment (danielroe) [inline packages/kit/src/logger.ts:7]:
```suggestion
  return tag ? logger.create(options).withTag(tag) : logger
```

**Pull request merged**